### DISCUSSION TITLE
BIRT installation dependency

### DISCUSSION THREAD

**Discussion by petethomas1** [eclipse-birt/birt#1212] [Q&A] [answered]
Hi,
I have an application (from a third party) I have been using for 6 or 7 years which is built on Eclipse 3.5. Now when I try to install it on a different workstation I get an error saying it requires 'org.eclipse.birt.feature.group [2.5.2,3.0.0)' but it could not be found.
Is there an archive site where I can find that package?

**Comment by wimjongman** [comment, marked as answer]
It should be in the archives. The location is on the front page of our website.
